Makefile

```diff
@@ -0,0 +1,8 @@
+build-docker:
+	docker build -t strapiv4 .
+
+run-docker:
```

Comment:
Could we remove docker-run.sh, or call docker-run.sh here? They seem to be doing the same thing.

Reply:
I split it out because if the image goes down, I don't want to do the building step again. Now that the dev dir is mounted, just running the container again is enough to get it back up with changes that were not there when the container was built.

Reply:
Oh I see. Maybe we can just remove docker-run.sh and document how to use the make commands in the README.